docs/_templates/autosummary/class.rst

```diff
@@ -5,9 +5,10 @@
 .. autoclass:: {{ objname }}
    :members:
    :show-inheritance:
-   {% if objname != "progress" %}
-      :inherited-members:
-   {% endif %}
+   {#- These classes inherit docstrings from the raw qt source, which generates rst syntax errors when building the docs #}
+   {% if objname not in ["progress", "cancelable_progress"] -%}
+   :inherited-members:
```

Comment:
Question only, does this need to be indented, or is that just style?

Reply:
Seems like style only!

Reply:
It can't be indented - that's what I had gotten wrong before. `:inherited-members:`  needs to have the same indentation level as `:show-inheritance:` and the other options, and we can't have any whitespace between these directive options (which is why the jinja template lines start and end with an extra `-` (this prevents extra whitespace from appearing in the final document)

Reply:
Ah thank you!